src/Moryx.Model.Sqlite/SqliteModelConfigurator.cs

```diff
@@ -72,7 +75,8 @@ public override DbContextOptions BuildDbContextOptions(IDatabaseConfig config)
 
         private static string BuildConnectionString(IDatabaseConfig config)
         {
-                return config.ConnectionSettings.ConnectionString;
+            var connectionStringBuilder = new SqliteConnectionStringBuilder(config.ConnectionSettings.ConnectionString);
+            return connectionStringBuilder.ToString();
```

Comment:
I'd assume, this should be `connectionStringBuilder.ConnectionString`. But fine, if this works.

Reply:
i will revert the chages here because i am not using it for "Pooling=False" anymore.